tests/manage/z_cluster/nodes/test_disk_failures.py

```diff
@@ -45,6 +47,19 @@ def finalizer():
             if not_ready_nodes:
                 nodes.restart_nodes(not_ready_nodes)
                 node.wait_for_nodes_status()
+
+            # Restart node if the osd stays at CLBO state
+            osd_pods_obj_list = get_osd_pods()
+            for pod in osd_pods_obj_list:
+                if pod.get().get(
+                    'status'
+                ).get(
+                    'containerStatuses'
+                )[0].get('state') == 'CrashLoopBackOff':
```

Comment:
Please add `CrashLoopBackOff` to constants.py under pod statuses

Reply:
Done